pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
The parameters here created some errors. Do you wanna change:
```suggestion
def divide(x, y):
```